google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BlobSnippets.java

```diff
@@ -173,6 +173,22 @@ public Blob copyToStrings(String bucketName, String blobName) {
   }
 
   /**
+   * Example of moving a blob to a different bucket with a different name.
+   */
+  public Blob moveTo(Blob srcBlob, String destBucket, String destBlob) {
```

Comment:
You should use the `blob` field instead of passing a blob parameter.

This method is also missing `TARGET` and `VARIABLE` tags:

```
// [TARGET copyTo(String, String, BlobSourceOption...)]
// [VARIABLE "my_unique_bucket"]
// [VARIABLE "copy_blob_name"]
```